python/cuml/fil/fil.pyx

```diff
@@ -288,26 +290,53 @@ class FIL(Base):
            loaded from a saved model using the treelite API
            https://treelite.readthedocs.io/en/latest/treelite-api.html
         output_class: boolean
-           True or False
-        algo : 0 = NAIVE, 1 = TREE_REORG, 2 = BATCH_TREE_REORG
+           If true, return a 1 or 0 depending on whether the raw prediction
+           exceeds the threshold. If False, just return the raw prediction.
+        algo : int (from algo_t enum)
+             0 (NAIVE) - simple inference using shared memory
+             1 (TREE_REORG) - similar to naive but trees rearranged to be more
+                              coalescing-friendly
+             2 (BATCH_TREE_REORG) - similar to TREE_REORG but predicting
+                                    multiple rows per thread block
         threshold : threshold is used to for classification
-           if output == OUTPUT_CLASS, else it is ignored
+           applied if output == OUTPUT_CLASS, else it is ignored
```

Comment:
Not sure what `output == OUTPUT_CLASS` means?

Reply:
This comment must be fixed to "if `output_class==True`".

Also, please drop `output_t` (see the comment above), we don't really need it for treelite.